requirements.txt

```diff
@@ -21,3 +21,4 @@ frozendict
 anytree>=2.4.3
 pyrevolve>=2.1.3
 distributed>=1.27
+git+https://github.com/devitocodes/devitobench
```

Comment:
I don't think DevitoBench should be a default requirement.

Reply:
It will probably not be. At the moment is just WIP-testing.
Related discussion happened under here: 
https://devitocodes.slack.com/archives/C7JMLMSG0/p1579101476448300

Reply:
still WIP?

Reply:
just a bit more

Reply:
this should be an optional dependence

if you use the mode `run`, instead of `bench`, you shouldn't need `devitobench` at all. That's what I would do, I think

Reply:
nonetheless, it should be an optional dependence, so be moved to a different file (@mloubout ?)

Reply:
Still not sure what to do with these optionals requirements